spring-data-microservice-transaction-sample/README.md

```diff
@@ -0,0 +1,548 @@
+spring.datasource.url=jdbc:scalardb:\
+?scalar.db.sql.connection_mode=direct\
+&scalar.db.storage=multi-storage\
+&scalar.db.multi_storage.storages=cassandra,mysql\
+&scalar.db.multi_storage.storages.cassandra.storage=cassandra\
+&scalar.db.multi_storage.storages.cassandra.contact_points=cassandra\
+&scalar.db.multi_storage.storages.cassandra.username=cassandra\
+&scalar.db.multi_storage.storages.cassandra.password=cassandra\
+&scalar.db.multi_storage.storages.mysql.storage=jdbc\
+&scalar.db.multi_storage.storages.mysql.contact_points=jdbc:mysql://mysql:3306/\
+&scalar.db.multi_storage.storages.mysql.username=root\
+&scalar.db.multi_storage.storages.mysql.password=mysql\
+&scalar.db.multi_storage.namespace_mapping=customer_service:mysql,order_service:cassandra,coordinator:cassandra\
+&scalar.db.multi_storage.default_storage=mysql\
+&scalar.db.sql.default_transaction_mode=two_phase_commit_transaction\
+&scalar.db.consensus_commit.isolation_level=SERIALIZABLE
+```
+
+- `scalar.db.sql.connection_mode`: This configuration decides how to connect to ScalarDB.
+- `scalar.db.storage`: Specifying `multi-storage` is necessary to use Multi-storage Transactions in ScalarDB.
+- `scalar.db.multi_storage.storages`: Your storage names must be defined here.
+- `scalar.db.multi_storage.storages.cassandra.*`: These configurations are for the `cassandra` storage, which is one of the storage names defined in `scalar.db.multi_storage.storages`. You can configure all the `scalar.db.*` properties for the `cassandra` storage here.
+- `scalar.db.multi_storage.storages.mysql.*`: These configurations are for the `mysql` storage, which is one of the storage names defined in `scalar.db.multi_storage.storages`. You can configure all the `scalar.db.*` properties for the `mysql` storage here.
+- `scalar.db.multi_storage.namespace_mapping`: This configuration maps the namespaces to the storage. In this sample application, operations for `customer_service` namespace tables are mapped to the `mysql` storage and operations for `order_service` namespace tables are mapped to the `cassandra` storage. You can also define which storage is mapped for the `coordinator` namespace that is used in Consensus Commit transactions.
+- `scalar.db.multi_storage.default_storage`: This configuration sets the default storage that is used for operations on unmapped namespace tables.
+- `scalar.db.sql.default_transaction_mode`: Specifying `two_phase_commit_transaction` is necessary to use Two-Phase Commit Transactions mode in ScalarDB.
+- `scalar.db.consensus_commit.isolation_level`: This configuration decides the isolation level used for ConsensusCommit.
+
+For details, please see [Configuration - Multi-storage Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/multi-storage-transactions.md#configuration).
+
+[The configuration for Order Service](order-service/src/main/resources/application.properties) is as follows:
```

Comment:
```suggestion
[The configuration for the Order Service](order-service/src/main/resources/application.properties) is as follows:
```

Reply:
Applying this change in https://github.com/scalar-labs/scalardb-samples/pull/40